react/components/FilterSidebar.js

```diff
@@ -127,7 +127,7 @@ class FilterSidebar extends Component {
             selectedFilters={selectedFilters}
             isOptionSelected={this.isOptionSelected}
           />
-          <div className="vtex-filter-buttons__box bt b--muted-5 bottom-0 fixed w-100 items-center flex">
+          <div className="vtex-filter-buttons__box bt b--muted-5 bottom-0 fixed w-100 items-center flex z-999 bg-base">
```

Comment:
`z-999` is used in very specific cases, I think that you should use another tachyons function.

Reply:
`z-1` did the job